.ci/jenkins_job_build.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+# Need the repo to know which tests to run.
+kata_repo="$1"
+
+tests_repo="github.com/kata-containers/tests"
+
+# This script is intended to execute under Jenkins
+# If we do not know where the Jenkins defined WORKSPACE area is
+# then quit
+if [ -z "${WORKSPACE}" ]
+then
+	echo "Jenkins WORKSPACE env var not set - exiting" >&2
+	exit 1
+fi
+
+# Put our go area into the Jenkins job WORKSPACE tree
+export GOPATH=${WORKSPACE}/go
+mkdir -p "${GOPATH}"
+
+# Export all environment variables needed.
+export GOROOT="/usr/local/go"
+export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:/sbin:${PATH}
+
+# Get the repository and move to the correct commit
+go get -d "${kata_repo}" || true
```

Comment:
I don't know if we want a `-u` on here as well, in case we are working in a dirty WORKSPACE, and the repo already exists. tbh, I don't know if `-d` implies `-u` on `go get` ?

Reply:
Agree